test/Game-test.js

```diff
@@ -1,8 +1,26 @@
 const { expect } = require('chai');
-const Game = require('../Game');
+const Game = require('../Game.js');
```

Comment:
Add a blank line between any third party libraries (like chai) that you are requiring and the files that you are bringing in:

```js
const { expect } = require('chai');

const Game = require('../Game.js');
const Puzzle = require('../Puzzle.js');
const Round = require('Round.js');
```